src/Main.java

```diff
@@ -9,7 +9,7 @@ public static void chekYear(int year) {
         }
     }
 
-    public static void deliveryCard(int deliveryDistance) {
+    public static int deliveryCard(int deliveryDistance) {
```

Comment:
лучше назвать с глагола и описать кратко, например, вот так
```suggestion
    public static int calculateDeliveryDays(int deliveryDistance) {
```

Reply:
Поправил